app/views/settings/cookie-policy.html.erb

```diff
@@ -76,6 +81,11 @@
               <td class="govuk-table__cell">Checks if you’ve visited Help for early years providers before. This helps us count how many people visit our site.</td>
               <td class="govuk-table__cell">24 hours</td>
             </tr>
+            <tr class="govuk-table__row">
+              <td class="govuk-table__cell">_gat_UA-147538067-8</td>
+              <td class="govuk-table__cell">Checks if you’ve used Help for early years providers before. This helps us count how many people visit our site.</td>
```

Comment:
The GA token looks a bit odd in the code, but then again its probably not going to change, and when it does the developer can find it here

Reply:
Yea, I looked at some other sites and it appears to be what others do.  If you can, please check from you machine and let me know it is the correct number.